frontend/src/Reducers/wishlistReducer.js

```diff
@@ -0,0 +1,47 @@
+const initialWishlistData = {
+  wishlistItems: JSON.parse(localStorage.getItem("maxStoreUserWishlist")) || [],
+};
+
+const wishlistReducer = (state, action) => {
+  switch (action.type) {
+    case "WISHLIST_ADD_ITEM":
```

Comment:
just a suggestion - this particular case can be done in 1 line using `.some()` and ternary operator `? :`.

Reply:
can you show?